src/prefect/tasks/monte_carlo/client.py

```diff
@@ -0,0 +1,256 @@
+            }
+            """,
+            variables=dict(
+                node_name=node_name,
+                object_id=object_id,
+                object_type=object_type,
+                resource_name=resource_name,
+                metadata_key=metadata_key,
+                metadata_value=metadata_value,
+            ),
+        )
+        return response["data"]["createOrUpdateLineageNode"]["node"]["mcon"]
+
+    def create_or_update_resource(self, resource_name: str, resource_type: str):
+        response = self.graphql(
+            query="""
+            mutation($resource_name: String!, $resource_type: String!) {
+              createOrUpdateResource(
+                isDefault: false,
+                name: $resource_name,
+                type: $resource_type,
+              ) {
+                resource {
+                  uuid
+                }
+              }
+            }
+            """,
+            variables=dict(resource_name=resource_name, resource_type=resource_type),
+        )
+        return response["data"]["createOrUpdateResource"]["resource"]["uuid"]
+
+    def create_or_update_lineage_edge(
+        self, source: dict, destination: dict, expire_at: str = None
+    ):
+        if expire_at is None:
+            expire_at = pendulum.now().add(days=1).isoformat()
+
+        response = self.graphql(
+            query="""
+                mutation($destination_object_id: String!,
+                $destination_object_type: String!,
+                $destination_resource_name: String!,
+                $source_object_id: String!, $source_object_type: String!,
+                 $source_resource_name: String!, $expire_at: DateTime) {
+                  createOrUpdateLineageEdge(
+                    destination: {
+                      objectId: $destination_object_id
+                      objectType: $destination_object_type
+                      resourceName: $destination_resource_name
+                    }
+                    source: {
+                      objectId: $source_object_id
+                      objectType: $source_object_type
+                      resourceName: $source_resource_name
+                    }
+                    expireAt: $expire_at
+                  ){
+                    edge{
+                      edgeId
+                    }
+                  }
+                }
+                """,
+            variables=dict(
+                destination_object_id=destination["object_id"],
+                destination_object_type=destination["object_type"],
+                destination_resource_name=destination["resource_name"],
+                source_object_id=source["object_id"],
+                source_object_type=source["object_type"],
+                source_resource_name=source["resource_name"],
+                expire_at=expire_at,
+            ),
+        )
+        self.logger.info("Response of createOrUpdateLineageEdge %s", response)
+        return response
```

Comment:
In other mutations you return just the `mcon`. Would it make sense for consistency sake to return just the `edge` from this function?